synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -346,16 +411,29 @@ export class SynopsysToolsParameter {
     }
 
     if (parseToBoolean(inputs.COVERITY_AUTOMATION_PRCOMMENT)) {
-      console.info("Coverity Automation comment is enabled");
-      covData.data.azure = await this.getAzureRepoInfo();
-      covData.data.environment = this.setEnvironmentScanPullData();
-      covData.data.coverity.automation.prcomment = true;
+      if (!isPullRequest()) {
+        taskLib.warning(
+          "Coverity PR comment is enabled but it will be ignored since it is not a PR/MR scan"
```

Comment:
Please align with the same warning message as action

Reply:
updated accordingly